src/main/java/backend/Logic.java

```diff
@@ -163,7 +164,7 @@ public void refresh() {
                     .thenRun(this::refreshUI)
                     .thenRun(() -> notifyRepoOpened(panel))
                     .thenCompose(n -> getRateLimitResetTime())
-                    .thenApply(this::updateRemainingRate)
+                    .thenAccept(this::updateApiQuotaInfoInGui)
                     .thenApply(rateLimits -> true)
```

Comment:
Since we use `thenAccept` instead of `thenApply`, the lambda's input parameter is no longer `rateLimits` (it's just void; the convention being used for this case is to call the input parameter `n`, like in the `thenCompose` call above).